### PR TITLE
stress_boot: fix boot up with mem and vcpus

### PR DESCRIPTION
**Pull request by yiqianwei**
1  Fix boot up with memory and vcpus
2. Set log into guest to 420s

ID: 2242

### CODE REVIEW

**Comment by yiqianwei** [comment]
Test results:
 (1/1) Host_RHEL.m9.u5.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2025.x86_64.io-github-autotest-qemu.stress_boot.q35: PASS (1414.15 s)

**Comment by yiqianwei** [comment]
@yanan-fu  and @jingzhao84 , Could you help to reivew this patch ? thanks

**Review by yanan-fu — APPROVED** [review]
Ack

File: generic/tests/cfg/stress_boot.cfg [hunk 1]
```diff
@@ -3,11 +3,14 @@
     type = stress_boot
     max_vms = 5
     alive_test_cmd = uname -a
-    login_timeout = 240
+    login_timeout = 420
```
Comment (jingzhao84) [inline generic/tests/cfg/stress_boot.cfg:6]: @yiqianwei 
Hello, why we set the value as "420"? do it the limitation value for the guest boot up?
Do you try all windows guest, all guest can be boot up successfully?

Thanks
Jing

Comment (yiqianwei) [reply generic/tests/cfg/stress_boot.cfg:6]: > @yiqianwei Hello, why we set the value as "420"? 
For rhel9.5.0, Need to long time on different host

do it the limitation value for the guest boot up? 
I think it's a time period,  set login vm with timeout time

Do you try all windows guest, all guest can be boot up successfully?
Yes, Run win11 guest, PASS
 (2/2) Host_RHEL.m10.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win11.x86_64.io-github-autotest-qemu.stress_boot.q35: PASS (523.07 s)

**Review by jingzhao84 — APPROVED** [review]

File: generic/tests/cfg/stress_boot.cfg (hunk 1 above)
Comment (jingzhao84) [reply generic/tests/cfg/stress_boot.cfg:6]: OK. understood

**Comment by jingzhao84** [comment]
Ack on my side

**Comment by yanan-fu** [comment]
Thanks all!

**Pull request merged**